public/locales/de/common.json

```diff
@@ -52,11 +52,11 @@
   "payWithGiroPay": "Spende mit GiroPay",
   "sepaDisclaimer": "SEPA-Zahlungen dauern 2-14 Werktage (in der Regel 3-5 Tage). Sobald sie eingegangen ist, senden wir dir eine Bestätigung per E-Mail mit deinem Baumzertifikat und erhöhen deinen Baumzähler.",
   "sepaMessage": "Indem du deine IBAN angibst und diese Zahlung bestätigst, ermächtigst du Plant-for-the-Planet und Stripe, unseren Zahlungsdienstleister, Anweisungen an deine Bank zu senden, um dein Konto zu belasten. Du hast Anspruch auf eine Rückerstattung durch deine Bank gemäß ihrer allgemeinen Geschäftsbedingungen. Wenn du irgendwelche Probleme mit deiner Spende hast, sende uns bitte eine E-Mail an support@plant-for-the-planet.org.",
-  "bankTransferDisclaimer": "Banküberweisungen benötigen in der Regel 1-5 Werktage, um uns zu erreichen. Sobald deine Überweisung eintrifft, senden wir dir eine Bestätigungs-E-Mail mit deinen Belegen (und deinem Zertifikat).",
+  "bankTransferDisclaimer": "Bitte überweise den Spendenbetrag auf unser Bankkonto. Die Zahlungsdetails zur automatischen Zuordnung erhältst du im nächsten Schritt. Sobald der Zahlungseingang verarbeitet wurde, schicken wir dir eine Bestätigungs-E-Mail, normalerweise dauert dies 2-5 Werktage. Übrigens: Deine SEPA-Überweisungen unterstützt zu 100% das Projekt, da keine Transaktionskosten entstehen.",
   "bankTransfer": "Banküberweisung",
   "continueWithBank": "Weiter mit Banküberweisung",
   "transferDetails": "Überweisungsdetails",
-  "transferDetailsMessage": "Vielen Dank! Um dein Spende zu vervollständigen, überweise sie bitte mit der unten angegebenen Referenz. Diese Informationen wurde dir auch per E-Mail zugesandt.",
+  "transferDetailsMessage": "Vielen Dank! Um deine Spende abzuschließen, überweise den Betrag bitte auf folgendes Bankkonto. Die Zahlungsdetails wurden dir zudem per E-Mail zugesandt. Gebe bitte den Verwendungszweck an, um eine automatische Zuordnung zu ermöglichen.",
```

Comment:
Same as above.

Reply:
You can also change this text in LingoHub and export it to GitHub to use our regular translation workflow.

Reply:
I don't have access to Lingohub, hence the pull request